common/content/services.js

```diff
@@ -16,7 +16,7 @@ const Services = Module("services", {
         this.jsm = window.Services;
         this.services = {
             "autoCompleteSearch": {
-                class_: "@mozilla.org/autocomplete/search;1?name=history",
+                class_: "@mozilla.org/autocomplete/search;1?name=" + (services.get("vc").compare(VERSION, "49") < 0 ? "history" : "unifiedcomplete"),
```

Comment:
@MoSal Can you show me what's the `:echo Services.appinfo.platformVersion` output?

@gkatsev Ok. I've updated it.